rust/src/smb/log.rs

```diff
@@ -129,61 +129,66 @@ fn smb_common_header(state: &SMBState, tx: &SMBTransaction) -> Json
     }
 
 
-    js.set_integer("session_id", tx.hdr.ssn_id);
-    js.set_integer("tree_id", tx.hdr.tree_id as u64);
+    jsb.set_uint("session_id", tx.hdr.ssn_id as u64)?;
+    jsb.set_uint("tree_id", tx.hdr.tree_id as u64)?;
 
-    debug_add_progress(&js, tx);
+    debug_add_progress(jsb, tx)?;
 
     match tx.type_data {
         Some(SMBTransactionTypeData::SESSIONSETUP(ref x)) => {
             if let Some(ref ntlmssp) = x.ntlmssp {
-                let jsd = Json::object();
+                let mut jsd = JsonBuilder::new_object();
```

Comment:
it is not necessary to create a new JsonBuilder here, just use:
`jsb.open_object("ntlmssp")?;` and when done with this json object use `jsb.close()?`.

Reply:
OK thanks.